litmus-portal/frontend/cypress/component/stepper.spec.tsx

```diff
@@ -1,20 +1,25 @@
 /// <reference types="Cypress" />
 
+import {
+  ApolloClient,
+  ApolloProvider,
+  InMemoryCache,
+} from '@apollo/client';
 import React from 'react';
 import { mount } from 'cypress-react-unit-test';
 import { Provider } from 'react-redux';
 import CustomStepper from '../../src/components/WorkflowStepper';
 import configureStore from '../../src/redux/configureStore';
+const {store} = configureStore();
 
-const { store } = configureStore();
+const client = new ApolloClient({
+  cache: new InMemoryCache(),
+});
+
+const wrapper = (<ApolloProvider client={client}><Provider store={store}><CustomStepper /></Provider></ ApolloProvider>);
```

Comment:
Why do you need Apollo here! Provider maybe for the theme but why Apollo!
Also could you change the name of `CustomStepper` to `WorkflowStepper`! That way it would be easier to identify components with the same name everywhere.

Reply:
Apollo provider is needed, because useMutation hook is used for inside the customStepper.
And yes the name, i can change.